src/test/java/com/iexec/core/task/TaskControllerTests.java

```diff
@@ -86,11 +87,29 @@ void setUp() {
     //region utilities
     @SneakyThrows
     String generateWalletAddress() {
-        ECKeyPair ecKeyPair = Keys.createEcKeyPair();
-        return Credentials.create(ecKeyPair).getAddress();
+        final ECKeyPair otherKeyPair = Keys.createEcKeyPair();
+        return Credentials.create(otherKeyPair).getAddress();
     }
     //endregion
 
+    // region deprecated methods, to remove in future version
+    @Test
+    void shouldCallGetTaskLogs() {
+        final TaskController controller = spy(new TaskController(challengeService, iexecHubService, replicatesService, taskLogsService, taskService));
```

Comment:
Is there a reason why you create a new `TaskController` instead of spying on the already-existing one?
```suggestion
        final TaskController controller = spy(taskController);
```

Reply:
fixed, thanks for the suggestion